include/envoy/network/address.h

```diff
@@ -91,6 +91,8 @@ class Ip {
    * @return the version of IP address.
    */
   virtual IpVersion version() const PURE;
+
+  virtual bool v6only() const PURE;
```

Comment:
Please add doc comments for what this means.

Reply:
Done!